libsolidity/formal/SMTEncoder.cpp

```diff
@@ -2705,6 +2682,40 @@ ModifierDefinition const* SMTEncoder::resolveModifierInvocation(ModifierInvocati
 	return modifier;
 }
 
+vector<FunctionDefinition const*> const& SMTEncoder::contractFunctions(ContractDefinition const& _contract)
+{
+	if (!m_contractFunctions.count(&_contract))
+	{
+		vector<FunctionDefinition const *> resolvedFunctions = _contract.definedFunctions();
+		for (auto const* base: _contract.annotation().linearizedBaseContracts)
+		{
+			if (base == &_contract)
+				continue;
+			for (auto const* baseFunction: base->definedFunctions())
+			{
+				if (baseFunction->isConstructor()) // We don't want to include constructors of parent contracts
+					continue;
```

Comment:
I have tried a version with base constructors and without and did not see any change in the tests.
My intuition is that constructors of parent contracts should not be included here.

Reply:
Agree